README.md

```diff
@@ -1,6 +1,72 @@
 # KWOK (Kube-WithOut-Kubelet)
 
-Simulate thousands of fake kubelets, on a laptop with minimum resource footprint.
+Kwok is a tool for simulating thousands of fake nodes.
+
+Kwokctl is a tool for creating fake clusters of nodes that are simulated by Kwok.
+
+## Examples
+
+These examples are tested to work with the latest released version of Kwok/Kwokctl.
+
+### Basic Usage
+
+- [Kwok Manage Nodes and Pods](./docs/examples/kwok-manage-nodes-and-pods.md) - Kwok's basic management of Node and Pod operations
+- Kwok - maintain Nodes heartbeat and Pods status.
+  - [Kwok in Cluster](./docs/examples/kwok-in-cluster.md) - Installing Kwok in a cluster
+  - [Kwok in Local](./docs/examples/kwok-in-local.md) - Run Kwok in the local for a cluster
+- Kwokctl - cluster creation, etcd snapshot, etc.
+  - [Kwokctl Manage Clusters](./docs/examples/kwokctl-manage-cluster.md) - Create/Delete a cluster in local where all nodes are managed by Kwok
+  - [Kwokctl Snapshot Clsuter](./docs/examples/kwokctl-snapshot.md) - Save/Restore the Etcd data of a cluster created by Kwokctl
+
+
+``` console
+$ time kwokctl create cluster
+Creating cluster "kwok-kwok"
+Starting cluster "kwok-kwok"
+Cluster "kwok-kwok" is ready
+You can now use your cluster with:
+
+    kubectl config use-context kwok-kwok
+
+Thanks for using kwok!
+
+real    0m2.599s
+user    0m0.606s
+sys     0m0.254s
+```
+
+### Actual Usage
```

Comment:
Not an advanced use case
Since Kwok is a new project, it would help to promote kwok if there is a friendly link with other projects (like https://github.com/kubernetes-sigs/external-dns#the-latest-release)

For advanced, we can later summarize the scenarios that users use more and then add

Reply:
so like "User Stories"?

Reply:
Yes, "User Stories" and "Friendly link"